style.css

```diff
@@ -1,3 +1,357 @@
+html {
+  box-sizing: border-box;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
+}
+
 body {
-  background: yellow;
+  margin: 0 auto;
+  font-family: "Source Sans Pro", sans-serif;
+  font-weight: 200px;
+}
+
+ul {
+  text-align: right;
+  list-style-type: none;
+  display: inline-block;
+}
+
+h1 {
+  margin: 0;
+  font-weight: 300;
+  font-size: 5rem;
+  text-align: center;
+}
+
+h4 {
+  margin: 0;
+  font-weight: 300;
+  font-size: 1rem;
+  text-align: center;
+  color: #303030;
+}
+
+h2, h3, h4 {
+  font-weight: 300;
+}
+
+.hero__container, {
```

Comment:
Remove the `,` after this or add a second valid selector. Right now this style is not being applied at all :(.